package.json

```diff
@@ -3,7 +3,6 @@
   "type": "module",
   "version": "3.0.20",
   "homepage": "https://elm-pages.com",
-  "moduleResolution": "node",
```

Comment:
This has never done anything. TS only reads from tsconfig.json.
It was introduced in 291378bd851e77affd172900788ff079be627020, which doesn't give any hints as to why this is here.